### ISSUE TITLE
externalPostgresql for Kutt

### ISSUE THREAD

**Issue by LordBilbon** [christianknell/helm-charts#740]
Hello, i try to use Kutt with a external Cloudsql Postgres but i have a issue : 

`Error: INSTALLATION FAILED: template: kutt/templates/postgresql-secret.yaml:9:107: executing "kutt/templates/postgresql-secret.yaml" at <b64enc>: invalid value; expected string`

I created a postgres secret before install.

Any idea ??

Thanks

**Comment by christianknell** [comment]
@LordBilbon Can you provide your `values.yaml` please.

**Comment by LordBilbon** [comment]
sure

```
postgresql:
  # -- enable PostgreSQL™ subchart from Bitnami
  enabled: false

externalPostgresql:
  auth:
    # -- Name of the database to use
    database: kutt
    # -- Name of existing secret to use for PostgreSQL credentials
    existingSecret: "postgres-kutt"
    # -- Password to use
    password: ""
    # -- Name of the user to use
    username: kutt
    # -- Key in the secret containing the password
    userPasswordKey: "password"
  # -- Hostname of the PostgreSQL database
  hostname: "<gcp-host>"
  # -- Port used to connect to PostgreSQL database
  port: 5432
```

**Comment by christianknell** [comment]
@LordBilbon should be fixed with #742. Please try with Helm Chart v2.4.4 again. And PLEASE never share secrets via Git. Their content is only base64-encoded and can easily be decoded to get the value of your passwords.

**Comment by LordBilbon** [comment]
It's works ;), thanks a lot
